src/host/outputStream.cpp

```diff
@@ -434,7 +432,7 @@ bool ConhostInternalGetSet::ResizeWindow(const size_t width, const size_t height
 
     // SetConsoleScreenBufferInfoEx however expects exclusive rects
     const auto sre = newViewport.ToExclusive();
-    csbiex.srWindow = sre;
+    csbiex.srWindow = til::unwrap_exclusive_small_rect(sre);

```

Comment:
so if i understand it correctly. the use of this API is a "code scent"?

Reply:
Yep! My intention is that this "interface" between "outside shorts" and "inside ints" is easily discoverable.